stellar/ttl.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const { Contract, SorobanRpc } = require('@stellar/stellar-sdk');
+const { Command } = require('commander');
+const { execSync } = require('child_process');
+const { loadConfig, printInfo, saveConfig } = require('../evm/utils');
+const { stellarCmd, getNetworkPassphrase, addBaseOptions } = require('./utils');
+const { getChainConfig, addOptionsToCommands } = require('../common');
+const { prompt } = require('../common/utils');
+require('./cli-utils');
+
+const MAX_INSTANCE_TTL_EXTENSION = 535679;
+
+async function getTtl(chain, contractName, _args, _options) {
+    printInfo('get ttl', contractName);
+    const ledgerEntry = await getLedgerEntry(chain, contractName);
+    printInfo('latest ledger', ledgerEntry.latestLedger);
+    printInfo('live until ledger', ledgerEntry.entries[0].liveUntilLedgerSeq);
```

Comment:
```suggestion
    printInfo('Latest ledger', ledgerEntry.latestLedger);
    printInfo('Expiry Ledger', ledgerEntry.entries[0].liveUntilLedgerSeq);
```
Capitalize

Reply:
add an estimate time as well? `(expiry - latest) * block time`

Reply:
> add an estimate time as well? `(expiry - latest) * block time`

Will do when revisiting this script following clarification from Stellar team about ledger times on local network.